app/ui/browser2/index.jsx

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Yeah probably, although I'd prefer to do that in a follow up since it'd be easy to do later